feconf.py

```diff
@@ -30,8 +31,9 @@
 # Whether we should serve the development or production experience.
 if PLATFORM == 'gae':
     DEV_MODE = (
-        not os.environ.get('SERVER_SOFTWARE')
-        or os.environ['SERVER_SOFTWARE'].startswith('Development'))
+        (not os.environ.get('SERVER_SOFTWARE')
+         or os.environ['SERVER_SOFTWARE'].startswith('Development'))
+        and os.environ.get('MINIFICATION') is None)
```

Comment:
Yes, it is the only way to do this. There is another thing we need to fix to simulate prod, we need to fix it here as well: https://github.com/oppia/oppia/blob/develop/appengine_config.py#L87. Otherwise numpy is not included and we get an error: https://gist.github.com/gvishal/6decc5ae62489244aecbb86ade7f5fdc